spec/lib/cli_spec.rb

```diff
@@ -1,4 +1,4 @@
-describe Rambo::CLI do
+RSpec.describe Rambo::CLI do
```

Comment:
Your test framework should not modify the code in which it is testing, with the exception of stubbing.

It does not mean much unless you turn on that config.

https://www.relishapp.com/rspec/rspec-core/v/3-0/docs/configuration/zero-monkey-patching-mode